redux/reducers/parameters/base/dotations.ts

```diff
@@ -83,7 +83,7 @@ export const BASE_DOTATIONS_DEFAULT_STATE: DotationsState = {
         popMinSeuilHaut: 10000,
         rapportPotentielFinancier: 2.5,
         pourcentageRangSeuilBas: 10,
-        pourcentageRangSeuilHaut: 66.6,
+        pourcentageRangSeuilHaut: 66.66,
```

Comment:
Améliorer l'arrondi correspondant au 2/3 ?
```suggestion
        pourcentageRangSeuilHaut: 66.67,
```